config/common.go

```diff
@@ -1,6 +1,7 @@
+//go:generate go run config_generator/main.go config.json
```

Comment:
Suggest we locate this directive in a file named `default.go` to tie it a bit more closely to `default.json` (and avoid putting it in a file that doesn't really deal with `default.json`).

Reply:
great idea!

Reply:
@reflog: ^?